XIPs/xip-17-remote-attachment-content-type-proposal.md

```diff
@@ -0,0 +1,74 @@
+---
+xip: 17
+title: Remote Attachment Content Type
+description: Content type for remote attachments
+author: Pat Nakajima
+status: Review
+type: Standards Track
+category: XRC
+created: 2023-02-15
+---
+
+## Abstract
+
+This XRC proposes a new content type for messages that do not consist of text and/or are too large to be sent on the XMTP network. It builds on [XIP 15](https://github.com/xmtp/XIPs/pull/15)'s basic attachment type, but instead of messages being sent on the network, the message contains a URL. The contents of the URL is an encrypted protobuf encoded `EncodedContent` that can be decrypted and decoded by clients.
+
+## Motivation
+
+The current 1MB limit for messages on the XMTP network is too small for many types of content. To enable larger files, external storage is going to be necessary.
+
+## Specification
+
+Proposed content type:
+
+```js
+{
+  authorityId: "xmtp.org"
+  typeId: "remoteAttachment"
```

Comment:
Thoughts on limiting this specification to the scope of StaticContent only?

Currently it is ambiguous whether the ContentServers can return Dynamic content. By explicitly only supporting static content then caching and Post-Delivery-storage approaches can avoid complications from cache Invalidation etc. 

Adding of the `contentDigest` mostly achieves this anyways, being explicit would make things cleaner

Reply:
What do you think of maybe `remoteStaticContent`?